Source/CombatExtended/Harmony/Harmony_StatWorker.cs

```diff
@@ -0,0 +1,89 @@
+using HarmonyLib;
+using Verse;
+using RimWorld;
+
+namespace CombatExtended.HarmonyCE
+{
+
+    [HarmonyPatch(typeof(StatWorker), nameof(StatWorker.ShouldShowFor))]
+    internal static class StatWorker_ShouldShowFor
+    {
+	internal static bool Prefix(ref bool __result, StatWorker __instance)
+	{
+	    if (!Controller.settings.ShowExtraStats) {
+		if (__instance.stat == global::CombatExtended.CE_StatDefOf.ToughnessRating)
+		{
+		    __result = false;
+		    return false;
+		}
+	    }
+	    return true;
+	}
+    }
+
+    [HarmonyPatch(typeof(StatWorker), nameof(StatWorker.GetStatDrawEntryLabel))]
```

Comment:
GetStatDrawEntryLabel is defined as `public virtual string GetStatDrawEntryLabel(StatDef stat, float value, ToStringNumberSense numberSense, StatRequest optionalReq, bool finalized = true)`, therefore is it not possible to modify StatWorkers/StatWorker_ArmorPartial.cs to show the exact same value?

Reply:
We could modify StatWorker_ArmorPartial, but I do not believe **that** is the stat worker at work here.  Let me check...  Or nevermind, it is using a custom StatWorker.